repo2docker/contentproviders/swhid.py

```diff
@@ -0,0 +1,113 @@
+import io
+import os
+import shutil
+import tarfile
+import time
+import re
+
+from os import path
+
+import requests
```

Comment:
We currently don't depend on `requests` and have used the standard library `urllib` to make HTTP requests. We should take a moment to review if we want to take on the additional maintenance cost of a new dependency vs sticking with using urllib.